src/analysis/processing/qgsalgorithmexporttospreadsheet.cpp

```diff
@@ -202,7 +202,7 @@ QVariantMap QgsExportToSpreadsheetAlgorithm::processAlgorithm( const QVariantMap
 #endif
   bool errored = false;
 
-  QgsProcessingMultiStepFeedback multiStepFeedback( mLayers.size(), feedback );
+  QgsProcessingMultiStepFeedback multiStepFeedback( static_cast<int>( mLayers.size() ), feedback );
```

Comment:
the steps argument in QgsProcessingMultiStepFeedback could be made std::size_t safely

Reply:
done